tools/osbuilder/rootfs-builder/cbl-mariner/Dockerfile.in

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Dallas Delaney, 2022 Microsoft Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG IMAGE_REGISTRY=mcr.microsoft.com
+FROM ${IMAGE_REGISTRY}/cbl-mariner/base/core:@OS_VERSION@
+
+RUN tdnf -y install \
+    ca-certificates \
+    curl \
+    g++ \
```

Comment:
In Ubuntu, `build-essential` includes `gcc`/`g++`/`make`, I'm not sure if `cbl-mariner`/`tdnf` is the same as ubuntu.